ci/infra/testrunner/README.md

```diff
@@ -122,6 +123,9 @@ optional arguments:
                         -v myconfig.yaml
   -p {openstack,vmware,bare-metal,libvirt}, --platform {openstack,vmware,bare-metal,libvirt}
                         The platform you're targeting. Defaults to openstack
+  -l {DEBUG,INFO,WARNING,ERROR}, --log-level {DEBUG,INFO,WARNING,ERROR}
```

Comment:
I get that this is consistent with `-p` above, but this is a weird way to document options.  Usually short and long options are next to each other, avoiding duplication of the option list.  Things like GNU utilities and Maria DB do it like this:
```suggestion
  -l, --log-level {DEBUG,INFO,WARNING,ERROR}
```
Probably goes in another PR, but I figured it might be worth mentioning...

Reply:
Ugh - this is Python's argparse formatting, isn't it?  So I guess that means messing with the help formatter...  So, definitely goes in a separate PR. :D
https://docs.python.org/3.6/library/argparse.html#formatter-class

Reply:
I agree the help formatting should be improved. I'll open an issue for that.